static/files/c2c/azure-metrics/example.json

```diff
@@ -0,0 +1,53 @@
+{
+    "api.version":"v1",
+    "source":{
+       "sourceType": "Polling",
+       "name": "azure-metrics-example",
+       "contentType": "AzureMetrics",
+       "thirdPartyRef":{
+          "resources":[
+             {
+                "serviceType": "AzureMetrics",
+                "path": {
+                   "type": "AzureMetricsPath",
+                   "environment": "Azure",
+                   "limitToRegions": ["westus2", "westeurope", "southeastasia"],
+                   "limitToNamespaces": ["Microsoft.Storage/storageAccounts", "Microsoft.Compute/virtualMachines", "Microsoft.EventHub/Namespaces"],
+                   "tagFilters":[
+                      {
+                          "namespace":"Microsoft.Storage/storageAccounts",
+                          "tags":[
+                              {
+                                  "name":"Cluster",
+                                  "values":["cloudcollector", "cocoa"]
+                              },
+                              {
+                                  "name":"Deployment",
+                                  "values":["prod"]
+                              }
+                          ]
+                      },
+                      {
+                          "namespace":"Microsoft.Compute/virtualMachines",
+                          "tags":[
+                              {
+                                  "name":"owner",
+                                  "values":["infra"]
+                              }
+                          ]
+                      }
+                  ]
+                },
+                "authentication": {
+                   "type": "AzureClientSecretAuthentication",
+                   "tenantId": "11111111-2222-3333-4444-555555555555",
+                   "clientId": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeeee",
+                   "clientSecret": "*****"
+                }
+             }
+          ]
+       },
+       "scanInterval": 300000,
+       "paused": false
```

Comment:
I vaguely remember that we discussed about the pause feature for azure metrics but forgot what the conclusion is 😅. If we haven't supported pause yet(or not tested for Beta), should we remove this from the example json, otherwise customer may set it to true? @samjsong

Reply:
Yes you're absolutely right, this was a mistake on my end. @JV0812 please remove the `paused` field from the example, thank you!

Reply:
Updated!